items/PARTY_HAT_CRAB.json

```diff
@@ -14,14 +11,15 @@
     "§7on your head for each SkyBlock",
     "§7year you\u0027ve been playing.",
     "",
-    "§7Your bonus: §b+0✎ Intelligence",
```

Comment:
Why did you remove this

Reply:
It doesn't exist anymore
I asked Nopo to give me the nbt data of the item when he put it on his head and it didn't show that anymore

Reply:
oh you are right nvm then